### ISSUE TITLE
Get answers on baseline [ROUND 0]

### ISSUE THREAD

**Issue by balthazarneveu** [balthazarneveu/molecule-retrieval-using-nlp#12]
With the right parameters and after 60 epochs of exp_8 + 60 epochs of exp_15 by resuming 8, we achieve 63% LRAP accuracy.
Seems like we're reaching a limit.
It could be useful to slightly keep pushing just to get our hands on how to get the right HP 

# Overfitting?
- [x] Do we really overfit? 
- _LRAP score validation : 63.335%_
- _LRAP score train : 76.666%_

the first impression may have been a numerical issue due to validation loss using batches. 
How to check? 
`python evaluation.py -e 15 -t` , :question: `-t` allows evaluating on training set

If it's not the case (not overfitting), can we simply move on to bigger models...

# can a HP search achieve better results?
- [x] Vary `batch_size` (30, 40, 50)

exp 30-39 batch_size=8 fits on T500 _& RTX2060_
exp 40-49 batch_size=16 fits on T500 _& RTX2060_
exp 50-59 batch_size=32 fits on RTX2060

- [x] Vary `lr` `1e-9, 1e-8, 1e-7, 1e-6, 2e-6` ADAM is said to be not too sensitive to LR ... this may be useless.

- [ ] Momentum in ADAM (betas)
